apps/messagegui/app.js

```diff
@@ -646,8 +646,18 @@ setTimeout(() => {
 }, 10); // if checkMessages wants to 'load', do that
 
 /* If the Bangle is unlocked by the user, treat that
-as a queue to stop repeated buzzing */
+as a queue to stop repeated buzzing.
```

Comment:
Minor English thing while we're here :)

```suggestion
as a cue to stop repeated buzzing.
```